app/views/layouts/application.html.haml

```diff
@@ -104,8 +104,8 @@
                   - @recent_letters.each do |l|
                     %tr
                       %td
-                        .thumbnail.pull-left
-                          = image_tag l.user.image, style: 'width:30px;'
+                        .thumbnail.pull-left{ style: "max-height: 50px; max-width: 50px" }
+                          = image_tag "https://res.cloudinary.com/#{ENV['CLOUDINARY_HANDLE']}/image/twitter_name/#{l.user.twitter_handle}.jpg",onerror: 'this.error=null;this.src="https://abs.twimg.com/sticky/default_profile_images/default_profile_normal.png"'
```

Comment:
Please create a helper in applciation_helper for this Twitter image tag. this helper should take the twitter handle as an option and generate this tag. This way you will have it called from index.html.haml and from here.